### PR TITLE
BUILD-522: shipwright-io/setup@v1

### PR DESCRIPTION
**Pull request by otaviof**
# Changes

Adopting shipwright-io/setup to prepare the environment for e2e tests, removed redundant scripts.

# Submitter Checklist

- [x] ~Includes tests if functionality changed/was added~
- [x] Includes docs if changes are user-facing
- [x] [Set a kind label on this PR](https://prow.k8s.io/command-help#kind)  
- [x] Release notes block has been filled in, or marked NONE

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/e2e.yaml [hunk 1]
```diff
@@ -18,45 +19,25 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Check out code
-        uses: actions/checkout@v3
+      - uses: actions/checkout@v3
         with:
           submodules: recursive
+      - uses: ./.github/actions/setup
```
Comment (HeavyWombat) [inline .github/workflows/e2e.yaml:25]: Nit: We could improve readability of the CI result page by giving this composite a name I think. What do you think?

image

```suggestion
      - name: Setup Go and ko
        uses: ./.github/actions/setup
```

**Review by HeavyWombat — CHANGES_REQUESTED** [review]
Awesome! Thank you for this initiative to simplify the setup and to make it usable in other repositories as well. I just have one remark where I would like to hear what you think.

**Review by SaschaSchwarze0 — APPROVED** [review]
/approve
/lgtm

**Pull request merged**